test/EVMHost.cpp

```diff
@@ -127,7 +125,10 @@ EVMHost::EVMHost(langutil::EVMVersion _evmVersion, evmc::VM& _vm):
 	else
 		assertThrow(false, Exception, "Unsupported EVM version");
 
-	tx_context.block_difficulty = evmc::uint256be{200000000};
+	if (m_evmRevision >= EVMC_PARIS)
+		tx_context.block_prev_randao = 0xa86c2e601b6c44eb4848f7d23d9df3113fbcac42041c49cbed5000cb4f118777_bytes32;
```

Comment:
This is the value from the merge block.

Reply:
It would probably be better to add that as a comment in the code.

Reply:
It doesn't actually matter what it is.

Reply:
Added.